src/frontend/apps/desk/src/app/InnerLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import { Loader } from '@openfun/cunningham-react';
+import { useEffect } from 'react';
+
+import useAuthStore from '@/auth/useAuthStore';
+import { Box } from '@/components';
+import { HEADER_HEIGHT, Header } from '@/features/header';
+import { Menu } from '@/features/menu';
+
+export default function InnerLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const { initAuth, authenticated, initialized } = useAuthStore();
+
+  useEffect(() => {
+    if (initialized) {
+      return;
+    }
+
+    initAuth();
+  }, [initAuth, initialized]);
```

Comment:
Suggestion: asap we should refactor this part, and better encapsulate login/logout logic. IMO this layout component should be dumb as possible.

Reply:
https://github.com/numerique-gouv/people/commit/3ebea72f3063879ee602a738e79e70210b5c1651